packages/yourloops/components/patient-data.tsx

```diff
@@ -85,27 +85,27 @@ function PatientDataPage(): JSX.Element | null {
   const userIsPatient = authHook.user?.isUserPatient();
   const prefixURL = userIsPatient ? "" : `/patient/${paramPatientId}`;
 
-  const initialized = authHook.isAuthHookInitialized && teamHook.initialized && blipApi !== null;
+  const initialized = authHook.isLoggedIn && teamHook.initialized && blipApi;
+
   React.useEffect(() => {
     if (!initialized) {
       return;
     }
 
     setTeams(teamHook.getMedicalTeams());
 
-
     if (userIsPatient && !_.isNil(authUser)) {
       setPatient(authUser);
     } else {
       const patientId = paramPatientId ?? userId;
-      if (patientId === null) {
+      if (!patientId) {
         log.error("Invalid patient Id", patientId);
         setError("Invalid patient Id");
         return;
       }
 
       const user = teamHook.getUser(patientId);
-      if (user === null || user.role !== UserRoles.patient) {
+      if (!user || user.role !== UserRoles.patient) {
```

Comment:
```
      if (!user || user.isPatient()) {
```

Reply:
Can't use `user.isPatient()` because getUser returns a TeamMember that extends Iuser, and not a User class instance. Weird...